### ISSUE TITLE
Replace KEY_HOME with KEY_HOMEPAGE

### ISSUE THREAD

**Issue by bmx666** [boundarydevices/linux-imx6#17]
In all kernels for Android starting from Kitkat in DTS gpio key **Home** assign as **KEY_HOME**, but must be **KEY_HOMEPAGE**

**Comment by gibsson** [comment]
Why do you say that? KEY_HOME is the right key code to trigger the HOME Intent:
http://androidxref.com/5.1.1_r6/xref/frameworks/base/data/keyboards/Generic.kl#124
http://androidxref.com/5.1.1_r6/xref/bionic/libc/kernel/uapi/linux/input.h#255

**Comment by bmx666** [comment]
According to [key mapping for Android](https://source.android.com/devices/input/keyboard-devices.html)
**KEY_HOME** is not **AC Home**
**KEY_HOMEPAGE** is **AC Home**

Example code from [imx6qdl-sabrelite.dtsi](https://github.com/boundarydevices/linux-imx6/blob/boundary-imx-l5.1.1_2.1.0-ga/arch/arm/boot/dts/imx6qdl-sabrelite.dtsi)

```
gpio-keys {
		...
		home {
			label = "Home";
			gpios = GP_GPIOKEY_HOME;
			linux,code = <KEY_HOME>;
		};
		...
```

must be

```
gpio-keys {
		...
		home {
			label = "Home";
			gpios = GP_GPIOKEY_HOME;
			linux,code = <KEY_HOMEPAGE>;
		};
		...
```

When you send input **KEY_HOME**, you just go to the first element of GUI list.
But logically like in smartphones and other Android devices, when you press "Home" button (**KEY_HOMEPAGE**), you go to launcher to default screen. It has more sense.

For example, see [Cyanogenmod commit](http://review.cyanogenmod.org/#/c/84816/)

**Comment by gibsson** [comment]
You're right, I'll fix it for the latest Android release (5.1.1). Thanks for bringing it up.

**Comment by gibsson** [comment]
Done, thanks again for your help!
